Mathlib/RingTheory/Coalgebra/TensorProduct.lean

```diff
@@ -136,6 +136,11 @@ instance instCoalgebra : Coalgebra R (A ⊗[R] B) where
     · dsimp
       simp only [one_smul]
 
+instance instIsCocomm [IsCocomm R A] [IsCocomm R B] : IsCocomm R (A ⊗[R] B) where
+  comm_comp_comul := by
+    ext : 1 -- wrong ext lemma!
+    sorry
```

Comment:
Eric, any idea how to make `ext` use the correct lemma here?

Reply:
Yes the one you want is not tagged ext, so `attribute [local ext]` it (and/or make a PR that re-tags it, since I think it was a lean3 performance decision)

Reply:
I don't really understand your request here, but I proved it for you.